pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -757,6 +764,14 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 		}
 	}
 
+	o.Logger.Info("Removing one-off annotations")
```

Comment:
```suggestion
	o.Logger.Info("Removing skip-readiness annotation")
```

I don't think we will have others, and if we can still refine this statement. For now, it's clearer to make it more concrete.
However, it would be nice if this is only logged when really done (to not pollute the log/confuse operators).